libsofia-sip-ua/nta/nta.c

```diff
@@ -1856,6 +1879,7 @@ int agent_get_params(nta_agent_t *agent, tagi_t *tags)
 	     NTATAG_TAG_3261(1),
 	     NTATAG_TIMEOUT_408(agent->sa_timeout_408),
 	     NTATAG_TIMER_C(agent->sa_timer_c),
+       NTATAG_TIMER_H(agent->sa_timer_h),
```

Comment:
Minor format difference, must be tabs-versus-space.

Reply:
My line of code is just pure spaces, those other line is combination of tabs and spaces. But will comply, just add 1 tab + 4 spaces